$OpenBSD: patch-configure,v 1.3 2015/10/08 09:41:10 jasper Exp $

Don't reset CFLAGS/LDFLAGS, which are needed to pass -L/usr/local/lib as
otherwise -ljpeg/-ltiff cannot be found.
Effectively reverts patch from https://bugzilla.gnome.org/show_bug.cgi?id=747982

--- configure.orig	Thu Oct  8 11:18:35 2015
+++ configure	Thu Oct  8 11:19:20 2015
@@ -20824,8 +20824,6 @@ fi
 if test "x$enable_jpeg" = "xyes"; then
 	SAVE_CFLAGS="$CFLAGS"
 	SAVE_LDFLAGS="$LDFLAGS"
-	CFLAGS=""
-	LDFLAGS=""
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for jpeg_destroy_decompress in -ljpeg" >&5
 $as_echo_n "checking for jpeg_destroy_decompress in -ljpeg... " >&6; }
 if ${ac_cv_lib_jpeg_jpeg_destroy_decompress+:} false; then :
@@ -20981,8 +20979,6 @@ fi
 if test x$enable_tiff = xyes ; then
 	SAVE_CFLAGS="$CFLAGS"
 	SAVE_LDFLAGS="$LDFLAGS"
-	CFLAGS=""
-	LDFLAGS=""
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for TIFFWriteScanline in -ltiff" >&5
 $as_echo_n "checking for TIFFWriteScanline in -ltiff... " >&6; }
 if ${ac_cv_lib_tiff_TIFFWriteScanline+:} false; then :
